app/upgrades/v9/accounts.go

```diff
@@ -0,0 +1,1816 @@
+package v9
+
+const MaxRecover = "73575669925896300000000"
```

Comment:
move to ./constants.go

Reply:
@ramacarlucho lets also change this to the exact amount and prove it in the tests.